book/http.md

```diff
@@ -234,8 +234,7 @@ This tells you that the host confirms that it, too, speaks `HTTP/1.0`,
 and that it found your request to be "OK" (which has a numeric code of
 200). You may be familiar with `404 Not Found`; that's another numeric
 code and response, as are `403 Forbidden` or `500 Server Error`. There
-are lots of these codes,^[As [this flow
-chart](https://github.com/for-GET/http-decision-diagram) shows.]
+are lots of these codes,
```

Comment:
I don't think this footnote is helpful. In a later "Go Further" we refer to Wikipedia page for a list of all codes.